README.md

```diff
@@ -1,37 +1,79 @@
-# Welcome
+Cloudinary URL-GEN SDK
+=========================
+[![Build Status](https://api.travis-ci.com/cloudinary/js-url-gen.svg?branch=master)](https://app.travis-ci.com/github/cloudinary/js-url-gen)
+## About
+The Cloudinary URL-GEN SDK allows you to quickly and easily integrate your application with Cloudinary.
+Effortlessly optimize and transform your cloud's assets.
 
-## About this project
+This SDK can also be used with [popular frontend frameworks](https://cloudinary.com/documentation/sdks/js/frontend-frameworks/index.html).
 
-This project enables you to create Cloudinary URLs for your images and videos.
-Using this SDK, you can apply advanced transformations to your images and videos.
+### Additional documentation
+This Readme provides basic installation and usage information.
+For the complete documentation, see the [URL-Gen SDK Guide](https://cloudinary.com/documentation/javascript_integration) and the [URL-GEN Reference](https://cloudinary.com/documentation/sdks/js/url-gen/index.html).
```

Comment:
```suggestion
For the complete documentation, see the [JavaScript SDK Guide](https://cloudinary.com/documentation/javascript_integration) and the [URL-GEN Reference](https://cloudinary.com/documentation/sdks/js/url-gen/index.html).
```

Reply:
@patrick-tolosa - if you're not going to change this to JavaScript, them can we at lease be consistent with Gen vs GEN?  So, either go with URL-Gen throughout, or URL-GEN throughout.